curriculum/challenges/english/25-front-end-development/lecture-working-with-common-string-methods/67326c3392068ec6184a0c95.md

```diff
@@ -150,15 +150,15 @@ If you call `"*".repeat(0);`, what is the output?
 
```

Comment:
can you remove the semicolon from right above here? it is not needed in a small snippet